src/Fantomas.Client/Contracts.fs

```diff
@@ -0,0 +1,87 @@
+module Fantomas.Client.Contracts
+
+open System
+open System.Collections.Generic
+open System.Threading
+
+[<RequireQualifiedAccess>]
+module Methods =
+    [<Literal>]
+    let Version = "fantomas/version"
+
+    [<Literal>]
+    let FormatDocument = "fantomas/formatDocument"
+
+    [<Literal>]
+    let FormatSelection = "fantomas/formatSelection"
+
+    [<Literal>]
+    let Configuration = "fantomas/configuration"
+
+type FormatDocumentRequest =
+    { SourceCode: string
+      /// File path will be used to identify the .editorconfig options
+      /// Unless the configuration is passed
+      FilePath: string
```

Comment:
Is this path expected to be the LSP-standard URI for file paths? https://microsoft.github.io/language-server-protocol/specification#uri

Reply:
No, not really actually as the client doesn't really know that there is an LSP thing behind the implementation.